tests/python/unittest/test_meta_schedule_relax_integration.py

```diff
@@ -0,0 +1,285 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This directory should not have relax tests

Reply:
Moved to the relax directory.